.github/workflows/build-builder.yaml

```diff
@@ -0,0 +1,35 @@
+name: build-builder
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
```

Comment:
you should specify the version and not latest as it may shift and provides unexpected behaviour

Reply:
:+1:

Reply:
done